Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -596,7 +596,14 @@ end
 				HasPluginChannel = { Params = "ChannelName", Return = "bool", Notes = "Returns true if the client has registered to receive messages on the specified plugin channel." },
 				IsUUIDOnline = { Params = "UUID", Return = "bool", IsStatic = true, Notes = "Returns true if the UUID is generated by online auth, false if it is an offline-generated UUID. We use Version-3 UUIDs for offline UUIDs, online UUIDs are Version-4, thus we can tell them apart. Accepts both 32-char and 36-char UUIDs (with and without dashes). If the string given is not a valid UUID, returns false."},
 				Kick = { Params = "Reason", Return = "", Notes = "Kicks the user with the specified reason" },
+				SendHideTitle = { Params = "", Return = "", Notes = "Sends to hide the title." },
```

Comment:
`Hides the title.` should be enough. I assume this does actually hide the title on the client. Does it hide the subtitle as well?